doc/python/selections.md

```diff
@@ -0,0 +1,315 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.7
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.9.0
+  plotly:
+    description: How to use selections in Python. Examples of adding and styling selections.
+    display_as: file_settings
+    language: python
+    layout: base
+    name: Selections
+    order: 26
+    permalink: python/selections/
+    thumbnail: thumbnail/make_selection.jpg
+---
+
+## Adding Selections to Cartesian Subplots
```

Comment:
Overall this document is great, but one thing which is missing is some indication to the reader of how to retrieve the selection values! We will need a link to some Dash documentation or something that folks can follow to actually do something with the selection once the user does it!

Do the selected values come back out using `fig.full_figure_for_development()` in a way that we can statically demo in the docs here?